fpga/src/main/scala/arty100t/HarnessBinders.scala

```diff
@@ -25,7 +25,17 @@ import testchipip._
 class WithArty100TUARTTSI extends HarnessBinder({
   case (th: HasHarnessInstantiators, port: UARTTSIPort) => {
     val ath = th.asInstanceOf[LazyRawModuleImp].wrapper.asInstanceOf[Arty100THarness]
-    ath.io_uart_bb.bundle <> port.io.uart
+    val harnessIO = IO(chiselTypeOf(port.io)).suggestName("uart_tsi")
+    harnessIO <> port.io
+    val packagePinsWithPackageIOs = Seq(
+      ("A9" , IOPin(harnessIO.uart.rxd)),
+      ("D10", IOPin(harnessIO.uart.txd)))
```

Comment:
Perhaps the simplest way to handle the bitstream generation failure is to tie 
`harnessIO.dropped` and `harnessIO.tsi2tl_state` to unused PMOD pins.

Reply:
What is surprising to me is that this PR does not touch those pins. And I have built bitstreams with those pins working in the past. 

This needs some investigation